packages/legend-query/src/components/QueryBuilderMilestoneEditor.tsx

```diff
@@ -122,7 +123,10 @@ const MilestoningParameterEditor = observer(
         .map((p) => buildElementOption(p) as PackageableElementOption<Type>);
     const changeType = (val: PackageableElementOption<Type>): void => {
       if (variableType !== val.value) {
-        milestoningParameter?.genericType?.value.setRawType(val.value);
+        const genType = milestoningParameter?.genericType?.value;
```

Comment:
I'm generally clueless about the strategy for `legend query` here regarding `mobx`.